code/modules/uplink/uplink_items/uplink_stealth.dm

```diff
@@ -65,6 +65,16 @@
 	surplus = 0
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
+/datum/uplink_item/stealthy_weapons/martialartsthree
+	name = "Krav Maga Scroll"
+	desc = "This scroll contains the secrets of an ancient martial arts technique. You will gain special unarmed attacks for \
+			stealthy takedowns."
+	item = /obj/item/book/granter/martial/krav_maga
+	cost = 16
+	player_minimum = 30
```

Comment:
Not sure what the player limit is on IAA, but given you have no super advantageous invisible defenses, beyond additional forms of attack, I don't think you'd need to add a player limit, unlike scarp. It is possible to take down someone with krav maga on your own.

Price seems fine though.

Reply:
IAA player req is 25 which is honestly close enough, so it's possible to remove the minimum for krav.